soifunc/deint.py

```diff
@@ -492,19 +513,20 @@ def SQTGMC(
                 ]
             )
         if denoiser == "dfttest":
-            dn_window = best_dfttest(
+            dn_window = DFTTest.denoise(
```

Comment:
What is the behavior of this without passing a `gpu` parameter? Will it default to using a GPU version if the system has a usable OpenCL device, or will it always use the CPU version?